wlms/ircbridge.go

```diff
@@ -150,6 +150,25 @@ func (bridge *IRCBridge) Connect(channels *IRCBridgerChannels) bool {
 			}
 		}
 	})
+	bridge.connection.AddCallback("*", func(e *irc.Event) {
+		// Wildcard event: Is triggered for every even
+		// Log the event and its data. Will probably create a lot of
+		// noise in the logfile but will hopefully tell me which
+		// command creates the IRC "ghosts" (no longer online IRC users,
+		// that are still listed on the metaserver)
+		switch e.Code {
```

Comment:
Have you tried how much data this produces? I do not know what sort of events we receive here, but if this is for everything that happens on IRC, could this overflow our disk on the server realistically?

Reply:
I have no idea and I don't know much about IRC anyway. I don't think it will request any messages we wouldn't receive anyway, so the amount of network traffic exchanged with the IRC server should be an upper limit. Based on that, I doubt that it can fill the disk, but it should be under observation while this is deployed.
Since the bug report describes a way to trigger the "ghost bug", I was considering deploying this branch, asking for someone to change the nickname a few times and hopefully create ghosts that way. Then I could look at the logs which IRC messages have to be handled additionally to avoid them.

Reply:
the repro plan sounds great. Also deploying and watching the disk space is fine. Just make sure that we are not flooding the disk, because a system with full disk is nearly unfixable - i.e. hard to ssh into and do anything.

Reply:
I deployed this branch two days ago and until now this logging didn't made any problems. It was logging a few PING/PONG messages (that I added to the ignore-list) and it is logging some messages when connecting to the IRC server, but nothing else yet. So until now nothing that fills the disk.